install/kubernetes/helm/subcharts/mixer/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if or (.Values.policy.enabled) (.Values.telemetry.enabled) }}
```

Comment:
could we change it to `{{- if ` and `{{- end` so it won't create a blank line?

otherwise, lgtm